client/elements/suttaplex/sc-suttaplex-list.html

```diff
@@ -238,6 +238,10 @@
               return this.suttaplexReturns ? (this.suttaplexReturns.length - index) : 0;
           }
 
+          _calculateOpened(suttaplexReturns) {
+              return (suttaplexReturns.length > 1) ? false : true;
```

Comment:
`return suttaplexReturns.length <= 1;`

Reply:
I made it <= 3. I figured that would include all the Abhidhamma texts from non-pali origin and would make more sense.